shared/fs/container.js

```diff
@@ -49,9 +49,12 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({
 })
 
 const mergeProps = (stateProps: StateProps, dispatchProps: DispatchProps, ownProps) => {
-  const pathItems = stateProps._itemNames.map(name =>
-    stateProps._pathItems.get(Types.pathConcat(stateProps.path, name), Constants.makeUnknownPathItem())
-  )
+  const pathItems = stateProps._itemNames
+    // TODO: hook up store when we have this in settings.
+    .filter(name => !name.startsWith('._') && name !== '.DS_Store' && name !== '.darwin')
```

Comment:
would it better to just have the daemon do this instead of the gui? just pass a flag to the list call to ignore files

Reply:
I was imagining a settings screen that allows user to toggle showing hidden files. In the future I suppose we'll also want to make it possible to hide files starting with `.`, i.e. the ones hidden by `ls` by default. So it seemed best to do it in UI.

Reply:
why does that setting matter, just make the call again right? we want the ui dumber

Reply:
Hmm we wouldn't need to make another RPC call if user doesn't navigate around in Files, and I thought just filtering in UI would be faster. I guess it wouldn't make a big difference.

Would you mind explaining why we want the UI dumber?

Reply:
i think the ui is harder to test and changes more often.  its harder for the ui to effectively cache. better to try and just make calls and pass flags vs doing this kind of stuff ourselves.